components/dashboard/src/settings/PersonalAccessTokens.tsx

```diff
@@ -534,7 +534,7 @@ function ListAccessTokensView() {
         <>
             <div className="flex items-center sm:justify-between mb-4">
                 <div>
-                    <h3>Personal Access Tokens</h3>
+                    <h3>Access Tokens</h3>
```

Comment:
**nitpick**: Few more. Last comment, promise.

https://github.com/gitpod-io/gitpod/blob/6386b4bcf484bfe87b8938580b8fe696c7ff0d46/components/dashboard/src/settings/PersonalAccessTokens.tsx#L340

https://github.com/gitpod-io/gitpod/blob/6386b4bcf484bfe87b8938580b8fe696c7ff0d46/components/dashboard/src/settings/PersonalAccessTokens.tsx#L413

https://github.com/gitpod-io/gitpod/blob/6386b4bcf484bfe87b8938580b8fe696c7ff0d46/components/dashboard/src/settings/PersonalAccessTokens.tsx#L654

https://github.com/gitpod-io/gitpod/blob/6386b4bcf484bfe87b8938580b8fe696c7ff0d46/components/dashboard/src/settings/PersonalAccessTokens.tsx#L666

Reply:
Interesting, looks like these must have resulted from a semantic merge as I removed them previously. Thanks for spotting, will get on these!

Reply:
updated